plugins/logitech-hidpp/fu-logitech-hidpp-peripheral.c

```diff
@@ -1021,7 +1021,4 @@ fu_logitech_hidpp_peripheral_init (FuLogitechHidPpPeripheral *self)
 	fu_device_set_protocol (FU_DEVICE (self), "com.logitech.unifying");
 	fu_device_set_version_format (FU_DEVICE (self), FWUPD_VERSION_FORMAT_PLAIN);
 
-	/* there are a lot of unifying peripherals, but not all respond
-	 * well to opening -- so limit to ones with issued updates */
```

Comment:
isn't this still valid? Some (early?) hidpp devices do not cope well being probed.

Reply:
I am not aware about any devices that do this, but I haven't dealt with that many early devices.

Reply:
I don't know, I was hoping to spur discussion to find out what devices are like this.  If it's just a handful, maybe we should just blacklist them.

What does solaar do for them?

Reply:
I think we ping everyone.

@hughsie do you remember which devices do this?

Reply:
I believe it was on advice from Marc/Nestor. I'd be fine switching to a blacklist approach if required.

Reply:
I think that's best. I think it's safe to say the big majority of devices don't have this issue. We can still query Logitech about this.

Reply:
OK if we merge this PR, i'm pretty sure logitech plugin was the only one using `FWUPD_DEVICE_FLAG_ONLY_SUPPORTED`.  I think we should leave it in the daemon for a release or two in case we gotta go back and then consider to tear it out later if not.

Reply:
Is this the kind of thing we want to put in a config file? e.g. a `logitech-hidpp.conf` with a `OnlySupported = false`? It does make me worry slightly about long term (read: difficult to update) distros like RHEL.

Reply:
That sounds good to me.  I'll add it.